Tasks/ArchiveFilesV2/archivefiles.ts

```diff
@@ -40,6 +40,9 @@ function getSevenZipLocation(): string {
 
 function findFiles(): string[] {
     if (includeRootFolder) {
+        if(!fs.existsSync(rootFolderOrFile)) {
+            tl.warning(`No file or directory found - ${rootFolderOrFile}`)
+        }
```

Comment:
Is this will be clear for the task users, what the warning about missing file/directory can lead to an ENOENT error?

Reply:
I think so,  ENOENT not verbose error occurs only in some specific cases. If the user doesn't understand where she comes from, a warning will help him.